docs/about/release-notes.rst

```diff
@@ -27,6 +27,8 @@ Features
   * The ``values`` property now returns a numpy array with ``ndim+1`` (vectors) or ``ndim+2``` (matrices) axes, with the inner 1 (vectors) or 2 (matrices) axes corresponding o the vector or matrix axes.
   * Vector or matrix element can now be accessed and modified directly using the new properties of ``Variable``, ``x1``, ``x2``, ``x3`` (for variables containing vectors) or ``x11``, ``x12``, ..., ``x33`` (for matrices).
 
+* Reduction operations such as ``sum`` and ``mean`` are now also multi-threaded and thus considerably faster `#1923 <https://github.com/scipp/scipp/pull/1923>`_.
```

Comment:
The benchmarks and other tests against `accumulate_in_place` show this working. At a later point it might be nice to demonstrate this in the `whats-new` notebook with a practical, previous slow running example. An equivalent of the 4d averaging used in the convolved smoothing for imaging for example. This can be done later in any case, rather than now.